### ISSUE TITLE
Match button color styles

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/aquila#106]
## Is your feature request related to a problem? Please describe.
We previously adjusted the `primary-button` class styles to fix the insufficient color contrast of Bootstrap's default styles (see PR #60). The other blue button (`btn-outline-primary`) we are using should match this color adjustment.

## Describe the solution you'd like
Add custom styles for `btn-outline-primary` class:

```
.btn-outline-primary {
  color: #0069d9;
  border-color: #0069d9;
}

.btn-outline-primary:hover {
  background-color: #0069d9;
  border-color: #0069d9;
}
```
